### PR TITLE
[14.0][FIX] product_manufacture: Keep manufacture information for first variant product creation

### PR DESCRIPTION
**Pull request by HviorForgeFlow**
Forward-Port of https://github.com/OCA/product-attribute/pull/930

CC @ForgeFlow @juppe @dreispt

### CODE REVIEW

**Review by MiquelRForgeFlow — APPROVED** [review]

File: product_manufacturer/models/product_manufacturer.py [hunk 1]
```diff
@@ -79,3 +79,23 @@ def _inverse_manufacturer_info(self):
...
                     template.manufacturer_purl
                 )
+
+    @api.model
+    def create(self, vals):
+        """Overwrite creation for rewriting manufacturer information (if set and having
+        only one variant), after the variant creation, that is performed in super.
+        """
+        template = super().create(vals)
+        if len(template.product_variant_ids) == 1:
```
Comment (MiquelRForgeFlow) [inline product_manufacturer/models/product_manufacturer.py]:
```suggestion
    @api.model_create_multi
    def create(self, vals_list):
        """Overwrite creation for rewriting manufacturer information (if set and having
        only one variant), after the variant creation, that is performed in super.
        """
        templates = super().create(vals_list)
        for template in templates:
            if len(template.product_variant_ids) == 1:
            (...)
```
😏

File: product_manufacturer/models/product_manufacturer.py [hunk 2]
```diff
@@ -79,3 +79,24 @@ def _inverse_manufacturer_info(self):
...
+                    related_vals["manufacturer"] = vals_list["manufacturer"]
+                if vals_list.get("manufacturer_pname"):
+                    related_vals["manufacturer_pname"] = vals_list["manufacturer_pname"]
+                if vals_list.get("manufacturer_pref"):
+                    related_vals["manufacturer_pref"] = vals_list["manufacturer_pref"]
+                if vals_list.get("manufacturer_purl"):
+                    related_vals["manufacturer_purl"] = vals_list["manufacturer_purl"]
+                if related_vals:
+                    template.write(related_vals)
+        return template
```
Comment (MiquelRForgeFlow) [inline product_manufacturer/models/product_manufacturer.py]:
```suggestion
        return templates
```

**Review by MiquelRForgeFlow — APPROVED** [review]

**Review by MiquelRForgeFlow — APPROVED** [review]

**Review by MiquelRForgeFlow — APPROVED** [review]

**Comment by MiquelRForgeFlow** [comment]
do rebase to solve ci

**Review by LoisRForgeFlow — APPROVED** [review]
LGTM

**Review by rousseldenis — APPROVED** [review]

**Comment by rousseldenis** [comment]
@HviorForgeFlow minor upgrade here?

**Pull request merged**